src/corelibs/U2Gui/src/options_panel/GroupOptionsWidget.h

```diff
@@ -69,7 +69,7 @@ class U2GUI_EXPORT GroupOptionsWidget : public QWidget {
     QVBoxLayout *mainLayout;
 
     static const int TITLE_HEIGHT = 30;
-    static const int MIN_WIDGET_WIDTH = 440;
+    static const int MIN_WIDGET_WIDTH = 480;  // Minimum for the tab of sequence viewer "PCR Primer Design".
```

Comment:
This will change width for all tabs in all views (assembly, sequence, msa, mca, tree)  regardless if pcr plugin is loaded or not.
This should be a dynamic per-tab variable set by the plugin.